src/node_sqlite.cc

```diff
@@ -1295,7 +1326,7 @@ void DatabaseSync::AggregateFunction(const FunctionCallbackInfo<Value>& args) {
       return;
     }
 
-    // Subract 1 because the first argument is the aggregate value.
+    // Subtract 1 because the first argument is the aggregate value.
```

Comment:
I'm guessing this change is related to https://github.com/nodejs/node/pull/56600#discussion_r2041207199. Please undo this and keep the PR limited to necessary changes only. You can open a separate PR for this.